instagram-app/src/components/CommentSection/CommentSection.jsx

```diff
@@ -1,19 +1,82 @@
-import React from 'react';
-import Comment from './Comment';
-import './CommentSection.css';
-
-let CommentSection = ({ comments }) => {
-  return (
-    <div className="comment-section">
-      {comments.map((comment, index) => {
-        return(
-          <div key={index}>
-            <Comment comment={comment} />
-          </div>
-        );
-      })}
-    </div>
-  )
+import React, { useState, useEffect } from "react";
+import Comment from "./Comment";
+import { Input } from "reactstrap";
+import "./CommentSection.css";
+
+let CommentSection = ({ parentId, comments, timeAgo }) => {
```

Comment:
Don't forget to use prop types every time one of your components takes in props.
Also, there is a reason you declared this as a let?